Explicitly define the Authorization Base URL (#165)
modelcontextprotocol/specification: docs/specification/draft/basic/authorization.md

For example: `MCP-Protocol-Version: 2024-11-05`

#### 2.3.2 Authorization Base URL

> The authorization base URL **MUST** be determined from the [SSE
> endpoint]({{< ref "specification/draft/basic/transports#http-with-sse" >}}) URL by
> preserving only the `scheme`, `hostname`, and `port` (if non-standard). For example:

Reviewer: ```suggestion
The authorization base URL **MUST** be determined from the [SSE
endpoint]({{< ref "specification/draft/basic/transports#http-with-sse" >}}) URL by discarding any existing `path` component. For example:
```
